bower.json

```diff
@@ -62,5 +62,8 @@
         "build/jsfeat.js"
       ]
     }
+  },
+  "resolutions": {
```

Comment:
Why is this in `<resolutions>` section and not in `<dependencies>`? Were there problems during installation?